warehouse/warehouse.go

```diff
@@ -625,6 +625,9 @@ func (wh *HandleT) createJobs(warehouse warehouseutils.WarehouseT) (err error) {
 		// If it is present do nothing else delete it
 		if _, inProgess := wh.isUploadJobInProgress(warehouse, uploadID); !inProgess {
 			wh.deleteWaitingUploadJob(uploadID)
+		} else {
+			// reset priority if the upload is already in progress
+			priority = 0
```

Comment:
Do we need to set this value to default (100) for resetting? Because processor queries for ascending order while picking up the JOB.